core/tests/protractor_utils/forms.js

```diff
@@ -406,8 +423,9 @@ var MultiSelectEditor = function(elem) {
     },
     expectCurrentSelectionToBe: async function(expectedCurrentSelection) {
       // Open the dropdown menu.
-      await elem.element(by.css(
-        '.protractor-test-search-bar-dropdown-toggle')).click();
+      var dropDownToggleButton = elem.element(
+        by.css('.protractor-test-search-bar-dropdown-toggle'));
+      await action.click('Test Dropdown Toggle', dropDownToggleButton);
 
       // Find the selected elements.
       var actualSelection = await elem.element(
```

Comment:
Can we wait for `selectedElem` to be visible before getting its text below?

Reply:
updated!